deploy/crs/v2/kieapp_rhdm_authoring-ha.yaml

```diff
@@ -8,3 +8,16 @@ metadata:
     consoleDesc: Deploys a DM Authoring HA environment
 spec:
   environment: rhdm-authoring-ha
+  objects:
```

Comment:
why instead doing it on every env.yaml file you don't do this at code lvl? it would save you a few lines and is better imho.
like here for example: https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kieapp/constants/constants.go

Or here: https://github.com/kiegroup/kie-cloud-operator/blob/81534c293f1df47d19113f5e6e749ccc080d2b7a/config/7.8.0/common.yaml#L43

@tchughesiv what do you suggest in this case?

Reply:
Agree... @desmax74 and I have discussed this same thing.